source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -155,52 +166,53 @@ A tie occurs when two or more documents have identical values in the
 field you are using to sort your results. MongoDB does not guarantee
 order if ties occur. 
 
-For example, in the sample data, there is a tie for the ``rating`` in
+For example, in the sample data, there is a tie for ``enrollment`` in
 the following documents:
 
 .. code-block:: none
    :copyable: false
 
-   [{_id ObjectID("...")} {type Assam} {rating 5}]
-   [{_id ObjectID("...")} {type English Breakfast} {rating 5}]
+   {"Title":"World Fiction","Enrollment":35}
+   {"Title":"Plate Tectonics","Enrollment":35}
 
 To guarantee a specific order for documents when ties occur, specify
 additional fields to sort by.
```

Comment:
Suggestion: 

This is not technically accurate. The documents could be tied on additional fields. Perhaps make the wording more general, something like "You can sort on additional fields to sort the documents that were tied in the previous sort."